src/error_diffusion.jl

```diff
@@ -183,3 +182,67 @@ Error diffusion algorithm using the filter
 ```
 """
 Atkinson() = ErrorDiffusion(OffsetMatrix([0 0 1 1; 1 1 1 0; 0 1 0 0]//8, 0:2, -1:2))
+
+"""
+    Fan()
+
+Error diffusion algorithm using the filter
+```
+      *  7
+1  3  5            (1//16)
+```
+A modification of the weights used in the Floyd-Steinberg algorithm.
+
+[1] Z. Fan, "A Simple Modification of Error Diffusion Weights",
+    IS&T's 46th Annual Conference, May 9-14, 1993, Final Program and Advanced Printing of
+    Paper Summaries, pp 113-115 (1993).
+"""
+Fan() = ErrorDiffusion(OffsetMatrix([0 0 0 7; 1 3 5 0]//16, 0:1, -2:1))
```

Comment:
The name `Fan` can be very ambiguous and is very likely to cause name conflict when used together with other ecosystems. Maybe `Fan93`?

Reply:
Good point!